src/app/ipv/middleware.js

```diff
@@ -187,7 +192,31 @@ module.exports = {
   },
   handleJourneyPage: async (req, res, next) => {
     try {
+      const currentEnvironment = process.env.NODE_ENV;
       const { pageId } = req.params;
+
+      if (
+        currentEnvironment === "development" &&
+        req.session.visitedAllTemplates
+      ) {
+        if (pageId === "page-ipv-reuse") {
+          let userDetailsResponse = samplePersistedUserDetails;
+          const i18n = req.i18n;
+          const userDetails = generateUserDetails(userDetailsResponse, i18n);
```

Comment:
Do we lose much readability by just having `const userDetails = generateUserDetails(samplePersistedUserDetails, req.i18n);` here and removing the above two lines?